### ISSUE TITLE
libdwarf does not build anymore

### ISSUE THREAD

**Issue by Max-Might** [haikuports/haikuports#414]
I was trying to run `hp libdwarf_x86` and the build stopped.
It seems like there is a conflict between `elf.h` and `libelf.h`.

```
checking elf.h usability... yes
checking elf.h presence... yes
checking for elf.h... yes
checking elfaccess.h usability... no
checking elfaccess.h presence... no
checking for elfaccess.h... no
checking libelf.h usability... yes
checking libelf.h presence... yes
checking for libelf.h... yes
checking libelf/libelf.h usability... yes
checking libelf/libelf.h presence... yes
checking for libelf/libelf.h... yes
checking for sys/types.h... (cached) yes
checking sys/ia64/elf.h usability... no
checking sys/ia64/elf.h presence... no
checking for sys/ia64/elf.h... no
Assuming struct Elf for the default libdwarf.h
checking for elf64_getehdr in -lelf... yes
checking for elf64_getshdr in -lelf... yes
libelf/libelf.h does not have struct _Elf
libelf.h does not have struct _Elf
configure: creating ./config.status
config.status: creating Makefile
config.status: creating config.h
gcc  -g -O2 -I. -I. -fPIC    -c -o dwarf_elf_access.o dwarf_elf_access.c
In file included from /boot/system/develop/headers/x86/libelf/sys_elf.h:75:0,
                 from /boot/system/develop/headers/x86/libelf/libelf.h:31,
                 from /boot/system/develop/headers/x86/libelf.h:1,
                 from dwarf_elf_access.c:57:
/boot/system/develop/headers/x86/libelf/elf_repl.h:36:25: error: conflicting types for 'Elf32_Addr'
 typedef __libelf_u32_t  Elf32_Addr;
                         ^
In file included from dwarf_incl.h:57:0,
                 from dwarf_elf_access.c:41:
/boot/system/develop/headers/os/kernel/elf.h:13:16: note: previous declaration of 'Elf32_Addr' was here
 typedef uint32 Elf32_Addr;
                ^
In file included from /boot/system/develop/headers/x86/libelf/sys_elf.h:75:0,
                 from /boot/system/develop/headers/x86/libelf/libelf.h:31,
                 from /boot/system/develop/headers/x86/libelf.h:1,
                 from dwarf_elf_access.c:57:
/boot/system/develop/headers/x86/libelf/elf_repl.h:38:25: error: conflicting types for 'Elf32_Off'
 typedef __libelf_u32_t  Elf32_Off;
                         ^
In file included from dwarf_incl.h:57:0,
                 from dwarf_elf_access.c:41:
/boot/system/develop/headers/os/kernel/elf.h:15:16: note: previous declaration of 'Elf32_Off' was here
 typedef uint32 Elf32_Off;

```

**Comment by korli** [comment]
You should try to rebuild libelf (in your case libelf_x86) and bump the version.

**Comment by korli** [comment]
Fixed in a09e9b3040a74b9b1bdee8a705caa814f08669dd
